docs/synthetics/using-the-api.md

```diff
@@ -0,0 +1,777 @@
+            }
+        ],
+        "minDelayBetweenNotificationsInMinutes": "10",
+        "backToNormalNeeded": true,
+        "failedRunCountToAlert": 1,
+        "notificationsEnabled": true,
+        "useOnlyAlertRuleIntegrations": false
+    }
+}'
+```
+
+To create a Browser Monitor to monitor a website, we would send a HTTP request as follows:
+```
+curl -L -X POST 'https://apps.sematext.com/synthetics-api/api/apps/17174/monitors/browser' \
+-H 'Authorization: apiKey 9bddb0a6-xxxx-xxxx-xxxx-397d15806cfd' \
+-H 'Content-Type: application/json' \
+--data-raw '{
+    "name": "Example browser monitor without a User Journey script",
+    "interval": "10m",
+    "enabled": true,
+    "locations": [
+        1,
+        2
+    ],
+    "url": "https://www.google.com/",
+    "script": "",
+    "scriptBased": false,
+    "conditions": [
+        {
+            "id": 1,
+            "type": "ERROR",
+            "operator": "=",
+            "value": "",
+            "enabled": true
+        },
+        {
+            "id": 2,
+            "type": "METRIC",
+            "key": "synthetics.time.response",
+            "operator": "<",
+            "value": "20000",
+            "enabled": true
+        }
+    ],
+    "alertRule": {
+        "schedule": [
+            {
+                "day": "Monday",
+                "index": 2,
+                "label": "MON",
+                "intervals": [],
+                "type": "ACTIVE"
+            },
+            {
+                "day": "Tuesday",
+                "index": 3,
+                "label": "TUE",
+                "intervals": [],
+                "type": "ACTIVE"
+            },
+            {
+                "day": "Wednesday",
+                "index": 4,
+                "label": "WED",
+                "intervals": [],
+                "type": "ACTIVE"
+            },
+            {
+                "day": "Thursday",
+                "index": 5,
+                "label": "THU",
+                "intervals": [],
+                "type": "ACTIVE"
+            },
+            {
+                "day": "Friday",
+                "index": 6,
+                "label": "FRI",
+                "intervals": [],
+                "type": "ACTIVE"
+            },
+            {
+                "day": "Saturday",
+                "index": 7,
+                "label": "SAT",
+                "intervals": [],
+                "type": "ACTIVE"
+            },
+            {
+                "day": "Sunday",
+                "index": 1,
+                "label": "SUN",
+                "intervals": [],
+                "type": "ACTIVE"
+            }
+        ],
+        "minDelayBetweenNotificationsInMinutes": "10",
+        "backToNormalNeeded": true,
+        "failedRunCountToAlert": 1,
+        "notificationsEnabled": true,
+        "useOnlyAlertRuleIntegrations": false
+    }
+}'
+```
+
+### Create a HTTP Monitor
+To create a HTTP Monitor, we would send a HTTP request as follows:
```

Comment:
a HTTP Monitor ==> an HTTP Monitor

Reply:
I apologize for this one. This one depends on which variant of English you speak and whether you pronounce the letter "H" as "aytch" or "haytch". I am trying to Americanize my English grammar as much as possible. For example "apologize" is spelled "apologise" in other non-American English speaking countries. This "an" error was one that slipped through the net, please forgive me for this one :)

Reply:
Ah, I was curious!  Thanks for explaining this.  I got learned ;)